docs/content/dagster-cloud/account/authentication/okta/scim-provisioning.mdx

```diff
@@ -0,0 +1,185 @@
+
+3. Click the Dagster Cloud app. **Note**: If you haven't set up SSO for Okta, [follow this guide](/dagster-cloud/account/authentication/setting-up-okta-saml-sso)) to do so before continuing.
+
+4. Click the **Sign On** tab and complete the following:
+
+   1. Click **Edit**.
+
+   2. In the **Advanced Sign-on Settings** section, enter the name of your organization in the **Organization** field.
+
+   3. In the **Credential Details** section, set the **Application username format** field to **Email**:
+
+      <Image
+      alt="Configured Sign On tab of Dagster Cloud Okta application"
+      src="/images/dagster-cloud/sso/okta/scim-sign-on-tab.png"
+      width={800}
+      height={606}
+      />
+
+   4. Click **Save**.
+
+5. Click the **Provisioning** tab and complete the following:
+
+   1. Click **Configure API Integration**.
+
+   2. Check the **Enable API integration** checkbox that displays.
+
+   3. In the **API Token** field, paste the Dagster Cloud API token you generated in [Step 1](#step-1-enable-scim-provisioning-in-dagster-cloud):
+
+      <Image
+      alt="Configured Provisioning tab of Dagster Cloud Okta application"
+      src="/images/dagster-cloud/sso/okta/provisioning-tab.png"
+      width={1024}
+      height={649}
+      />
+
+   4. Click **Test API Credentials** to verify that your organization and API token work correctly.
+
+   5. When finished, click **Save**.
+
+---
+
+## Step 3: Enable user syncing in Okta
+
+After you confirm that your API credentials work in the Dagster Cloud Okta application, you can enable user syncing:
+
+1. In the Dagster Cloud Okta app, click the **Provisioning** tab.
+
+2. In the **Settings** panel, click **To App**.
+
+3. Click **Edit**.
+
+4. Next to **Create Users**, check the **Enable** checkbox. **Note**: The default username used to create accounts must be set to **Email** or user provisioning may not work correctly:
+
+   <Image
+   alt="Highlighted Create users setting and default username setting in Okta"
+   src="/images/dagster-cloud/sso/okta/provisioning-to-app-create-users.png"
+   width={1576}
+   height={380}
+   />
+
+5. Optionally, check **Enable** next to **Update User Attributes** and **Deactivate Users** to enable these features.
+
+6. When finished, click **Save**.
+
+---
+
+## Step 4: Enable group syncing in Okta
+
+<Note>
+  This step is required only if you want to sync Okta user groups to Dagster
+  Cloud as{" "}
+  <a href="/dagster-cloud/account/managing-users/managing-teams">Teams</a>.
+</Note>
+
+When **Push groups** is enabled in Okta, you can sync user groups from Okta to Dagster Cloud as [Teams](/dagster-cloud/account/managing-users/managing-teams). Refer to the [Okta documentation](https://help.okta.com/oie/en-us/Content/Topics/users-groups-profiles/usgp-enable-group-push.htm) for setup instructions.
+
+---
+
+## Step 5: TODO
+
+TODO: Are there further steps / testing users should do to set this up?
+
```

Comment:
this could be a good place to say things like:
- once okta syncs, users in the users page will have a little icon next to them
- You can no longer add new users from the Dagster Cloud UI, do it in the iDP
- You can only remove users who have not been synced
- You can still change permissions and make changes to teams 
- you can turn off SCIM syncing at any time from the Dagster Cloud UI and it will leave the users the way they were when you turned it off

Looks like all or most of that is on the other page, so I think we could link to that?